test/benchee/formatters/csv_integration_test.exs

```diff
@@ -1,36 +1,35 @@
 defmodule Benchee.Formatters.CSVIntegrationTest do
-  use ExUnit.Case
+  use ExUnit.Case, async: true
```

Comment:
this test does IO, I don't think it should be run async? :thinking:

Reply:
I mean, we could do it, but it's not worth it. I'll remove.